[DEBUG][2025-03-02 09:10:51][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:10:51][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:10:51][config.py:267] - [Config] User datas loaded.
[INFO][2025-03-02 09:10:51][app.py:371] - Assets path configured: D:\Program\dify-on-wechat\assets
[INFO][2025-03-02 09:10:51][flet_socket_server.py:61] - Starting up TCP server on localhost:64052
[INFO][2025-03-02 09:10:51][app.py:171] - Flet app has started...
[INFO][2025-03-02 09:10:51][app.py:131] - App URL: tcp://localhost:64052
[INFO][2025-03-02 09:10:51][__init__.py:62] - Starting Flet View app...
[INFO][2025-03-02 09:10:51][__init__.py:86] - Looking for Flet executable at: D:\Program\0-venv\dify-on-wechat-env\Lib\site-packages\flet_desktop\app\flet\flet.exe
[INFO][2025-03-02 09:10:51][__init__.py:88] - Flet View found in: D:\Program\0-venv\dify-on-wechat-env\Lib\site-packages\flet_desktop\app\flet\flet.exe
[INFO][2025-03-02 09:10:51][app.py:240] - App session started
[DEBUG][2025-03-02 09:10:51][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:10:51][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:10:51][config.py:267] - [Config] User datas loaded.
[INFO][2025-03-02 09:10:57][config.py:353] - [Config] Configuration saved.
[DEBUG][2025-03-02 09:11:05][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:11:05][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:11:05][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:11:05][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:11:05][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:11:05][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:11:05][audio_convert.py:10] - import pysilk failed, wechaty voice message will not be supported.
[WARNING][2025-03-02 09:11:05][audio_convert.py:20] - import pilk failed, silk voice conversion will not be supported. Try: pip install pilk
[INFO][2025-03-02 09:11:05][gewechat_channel.py:62] - [gewechat] init: base_url: http://192.168.31.235:2531/v2/api, token: 934c7ad0b1994c13a52374f5be9f39fb, app_id: wx_h_wNG48atr8kFFQWkELQ2, download_url: http://192.168.31.235:2532/download
[INFO][2025-03-02 09:11:05][plugin_manager.py:50] - Loading plugins config...
[INFO][2025-03-02 09:11:05][plugin_manager.py:88] - Scaning plugins ...
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Banwords_v1.0 registered, path=./plugins\banwords
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin BDunit_v0.1 registered, path=./plugins\bdunit
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin CustomDifyApp_v0.2 registered, path=./plugins\custom_dify_app
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Dungeon_v1.0 registered, path=./plugins\dungeon
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Finish_v1.0 registered, path=./plugins\finish
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Godcmd_v1.0 registered, path=./plugins\godcmd
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Hello_v0.1 registered, path=./plugins\hello
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin JinaSum_v0.0.1 registered, path=./plugins\jina_sum
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Keyword_v0.1 registered, path=./plugins\keyword
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin linkai_v0.1.0 registered, path=./plugins\linkai
[INFO][2025-03-02 09:11:05][plugin_manager.py:41] - Plugin Role_v1.0 registered, path=./plugins\role
[WARNING][2025-03-02 09:11:05][plugin_manager.py:113] - Failed to import plugin tool: No module named 'chatgpt_tool_hub'
[DEBUG][2025-03-02 09:11:05][plugin_manager.py:185] - plugins.json config={'plugins': SortedDict({'Godcmd': {'enabled': True, 'priority': 999}, 'Keyword': {'enabled': True, 'priority': 900}, 'Banwords': {'enabled': False, 'priority': 100}, 'linkai': {'enabled': False, 'priority': 99}, 'JinaSum': {'enabled': False, 'priority': 10}, 'Role': {'enabled': False, 'priority': 0}, 'Dungeon': {'enabled': False, 'priority': 0}, 'CustomDifyApp': {'enabled': True, 'priority': 0}, 'BDunit': {'enabled': False, 'priority': 0}, 'Hello': {'enabled': True, 'priority': -1}, 'Finish': {'enabled': True, 'priority': -999}}, sort_func=<lambda>, reverse=True)}
[DEBUG][2025-03-02 09:11:05][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\godcmd\config.json, exist=True
[DEBUG][2025-03-02 09:11:05][plugin.py:28] - loading plugin config, plugin_name=Godcmd, conf={'password': '', 'admin_users': []}
[INFO][2025-03-02 09:11:05][godcmd.py:194] - [Godcmd] 因未设置口令，本次的临时口令为1985。
[INFO][2025-03-02 09:11:05][godcmd.py:210] - [Godcmd] inited
[DEBUG][2025-03-02 09:11:05][keyword.py:35] - [keyword]加载配置文件D:\Program\dify-on-wechat\plugins\keyword\config.json
[INFO][2025-03-02 09:11:05][keyword.py:41] - [keyword] {}
[INFO][2025-03-02 09:11:05][keyword.py:43] - [keyword] inited.
[DEBUG][2025-03-02 09:11:05][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\custom_dify_app\config.json, exist=False
[DEBUG][2025-03-02 09:11:05][plugin.py:28] - loading plugin config, plugin_name=CustomDifyApp, conf=None
[INFO][2025-03-02 09:11:05][custom_dify_app.py:25] - [CustomDifyApp] config is None
[DEBUG][2025-03-02 09:11:05][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\hello\config.json, exist=False
[DEBUG][2025-03-02 09:11:05][plugin.py:28] - loading plugin config, plugin_name=Hello, conf=None
[DEBUG][2025-03-02 09:11:05][hello.py:126] - No Hello plugin config.json, use plugins/hello/config.json.template
[INFO][2025-03-02 09:11:05][hello.py:38] - [Hello] inited
[INFO][2025-03-02 09:11:05][finish.py:23] - [Finish] inited
[INFO][2025-03-02 09:11:05][gewechat_channel.py:89] - [gewechat] sleep 3 seconds waiting for server to start, then set callback
[INFO][2025-03-02 09:11:05][gewechat_channel.py:107] - [gewechat] start callback server: http://192.168.31.235:9919/v2/api/callback/collect, using port 9919
[DEBUG][2025-03-02 09:11:08][gewechat_channel.py:229] - [gewechat] receive data: b'{"testMsg":"\xe5\x9b\x9e\xe8\xb0\x83\xe5\x9c\xb0\xe5\x9d\x80\xe9\x93\xbe\xe6\x8e\xa5\xe6\x88\x90\xe5\x8a\x9f\xef\xbc\x81","token":"934c7ad0b1994c13a52374f5be9f39fb"}'
[DEBUG][2025-03-02 09:11:08][gewechat_channel.py:234] - [gewechat] 收到gewechat服务发送的回调测试消息
[DEBUG][2025-03-02 09:11:46][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:11:46][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:11:46][config.py:267] - [Config] User datas loaded.
[INFO][2025-03-02 09:11:46][app.py:371] - Assets path configured: D:\Program\dify-on-wechat\assets
[INFO][2025-03-02 09:11:46][flet_socket_server.py:61] - Starting up TCP server on localhost:64121
[INFO][2025-03-02 09:11:46][app.py:171] - Flet app has started...
[INFO][2025-03-02 09:11:46][app.py:131] - App URL: tcp://localhost:64121
[INFO][2025-03-02 09:11:46][__init__.py:62] - Starting Flet View app...
[INFO][2025-03-02 09:11:46][__init__.py:86] - Looking for Flet executable at: D:\Program\0-venv\dify-on-wechat-env\Lib\site-packages\flet_desktop\app\flet\flet.exe
[INFO][2025-03-02 09:11:46][__init__.py:88] - Flet View found in: D:\Program\0-venv\dify-on-wechat-env\Lib\site-packages\flet_desktop\app\flet\flet.exe
[INFO][2025-03-02 09:11:46][app.py:240] - App session started
[DEBUG][2025-03-02 09:11:46][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:11:46][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:11:46][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:11:50][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:11:50][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:11:50][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:11:50][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:11:50][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:11:50][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:11:50][audio_convert.py:10] - import pysilk failed, wechaty voice message will not be supported.
[WARNING][2025-03-02 09:11:50][audio_convert.py:20] - import pilk failed, silk voice conversion will not be supported. Try: pip install pilk
[INFO][2025-03-02 09:11:50][gewechat_channel.py:62] - [gewechat] init: base_url: http://192.168.31.235:2531/v2/api, token: 934c7ad0b1994c13a52374f5be9f39fb, app_id: wx_h_wNG48atr8kFFQWkELQ2, download_url: http://192.168.31.235:2532/download
[INFO][2025-03-02 09:11:50][plugin_manager.py:50] - Loading plugins config...
[INFO][2025-03-02 09:11:50][plugin_manager.py:88] - Scaning plugins ...
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Banwords_v1.0 registered, path=./plugins\banwords
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin BDunit_v0.1 registered, path=./plugins\bdunit
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin CustomDifyApp_v0.2 registered, path=./plugins\custom_dify_app
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Dungeon_v1.0 registered, path=./plugins\dungeon
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Finish_v1.0 registered, path=./plugins\finish
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Godcmd_v1.0 registered, path=./plugins\godcmd
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Hello_v0.1 registered, path=./plugins\hello
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin JinaSum_v0.0.1 registered, path=./plugins\jina_sum
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Keyword_v0.1 registered, path=./plugins\keyword
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin linkai_v0.1.0 registered, path=./plugins\linkai
[INFO][2025-03-02 09:11:50][plugin_manager.py:41] - Plugin Role_v1.0 registered, path=./plugins\role
[WARNING][2025-03-02 09:11:50][plugin_manager.py:113] - Failed to import plugin tool: No module named 'chatgpt_tool_hub'
[DEBUG][2025-03-02 09:11:50][plugin_manager.py:185] - plugins.json config={'plugins': SortedDict({'Godcmd': {'enabled': True, 'priority': 999}, 'Keyword': {'enabled': True, 'priority': 900}, 'Banwords': {'enabled': False, 'priority': 100}, 'linkai': {'enabled': False, 'priority': 99}, 'JinaSum': {'enabled': False, 'priority': 10}, 'Role': {'enabled': False, 'priority': 0}, 'Dungeon': {'enabled': False, 'priority': 0}, 'CustomDifyApp': {'enabled': True, 'priority': 0}, 'BDunit': {'enabled': False, 'priority': 0}, 'Hello': {'enabled': True, 'priority': -1}, 'Finish': {'enabled': True, 'priority': -999}}, sort_func=<lambda>, reverse=True)}
[DEBUG][2025-03-02 09:11:50][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\godcmd\config.json, exist=True
[DEBUG][2025-03-02 09:11:50][plugin.py:28] - loading plugin config, plugin_name=Godcmd, conf={'password': '', 'admin_users': []}
[INFO][2025-03-02 09:11:50][godcmd.py:194] - [Godcmd] 因未设置口令，本次的临时口令为8941。
[INFO][2025-03-02 09:11:50][godcmd.py:210] - [Godcmd] inited
[DEBUG][2025-03-02 09:11:50][keyword.py:35] - [keyword]加载配置文件D:\Program\dify-on-wechat\plugins\keyword\config.json
[INFO][2025-03-02 09:11:50][keyword.py:41] - [keyword] {}
[INFO][2025-03-02 09:11:50][keyword.py:43] - [keyword] inited.
[DEBUG][2025-03-02 09:11:50][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\custom_dify_app\config.json, exist=False
[DEBUG][2025-03-02 09:11:50][plugin.py:28] - loading plugin config, plugin_name=CustomDifyApp, conf=None
[INFO][2025-03-02 09:11:50][custom_dify_app.py:25] - [CustomDifyApp] config is None
[DEBUG][2025-03-02 09:11:50][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\hello\config.json, exist=False
[DEBUG][2025-03-02 09:11:50][plugin.py:28] - loading plugin config, plugin_name=Hello, conf=None
[DEBUG][2025-03-02 09:11:50][hello.py:126] - No Hello plugin config.json, use plugins/hello/config.json.template
[INFO][2025-03-02 09:11:50][hello.py:38] - [Hello] inited
[INFO][2025-03-02 09:11:50][finish.py:23] - [Finish] inited
[INFO][2025-03-02 09:11:50][gewechat_channel.py:89] - [gewechat] sleep 3 seconds waiting for server to start, then set callback
[INFO][2025-03-02 09:11:50][gewechat_channel.py:107] - [gewechat] start callback server: http://192.168.31.235:9919/v2/api/callback/collect, using port 9919
[DEBUG][2025-03-02 09:11:53][gewechat_channel.py:229] - [gewechat] receive data: b'{"testMsg":"\xe5\x9b\x9e\xe8\xb0\x83\xe5\x9c\xb0\xe5\x9d\x80\xe9\x93\xbe\xe6\x8e\xa5\xe6\x88\x90\xe5\x8a\x9f\xef\xbc\x81","token":"934c7ad0b1994c13a52374f5be9f39fb"}'
[DEBUG][2025-03-02 09:11:53][gewechat_channel.py:234] - [gewechat] 收到gewechat服务发送的回调测试消息
[DEBUG][2025-03-02 09:12:09][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:12:09][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:12:09][config.py:267] - [Config] User datas loaded.
[INFO][2025-03-02 09:12:09][app.py:371] - Assets path configured: D:\Program\dify-on-wechat\assets
[INFO][2025-03-02 09:12:09][flet_socket_server.py:61] - Starting up TCP server on localhost:64143
[INFO][2025-03-02 09:12:09][app.py:171] - Flet app has started...
[INFO][2025-03-02 09:12:09][app.py:131] - App URL: tcp://localhost:64143
[INFO][2025-03-02 09:12:09][__init__.py:62] - Starting Flet View app...
[INFO][2025-03-02 09:12:09][__init__.py:86] - Looking for Flet executable at: D:\Program\0-venv\dify-on-wechat-env\Lib\site-packages\flet_desktop\app\flet\flet.exe
[INFO][2025-03-02 09:12:09][__init__.py:88] - Flet View found in: D:\Program\0-venv\dify-on-wechat-env\Lib\site-packages\flet_desktop\app\flet\flet.exe
[INFO][2025-03-02 09:12:10][app.py:240] - App session started
[DEBUG][2025-03-02 09:12:10][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:12:10][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:12:10][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:12:16][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:12:16][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:12:16][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:12:16][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:12:16][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:12:16][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:12:16][audio_convert.py:10] - import pysilk failed, wechaty voice message will not be supported.
[WARNING][2025-03-02 09:12:16][audio_convert.py:20] - import pilk failed, silk voice conversion will not be supported. Try: pip install pilk
[INFO][2025-03-02 09:12:16][gewechat_channel.py:62] - [gewechat] init: base_url: http://192.168.31.235:2531/v2/api, token: 934c7ad0b1994c13a52374f5be9f39fb, app_id: wx_h_wNG48atr8kFFQWkELQ2, download_url: http://192.168.31.235:2532/download
[INFO][2025-03-02 09:12:16][plugin_manager.py:50] - Loading plugins config...
[INFO][2025-03-02 09:12:16][plugin_manager.py:88] - Scaning plugins ...
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Banwords_v1.0 registered, path=./plugins\banwords
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin BDunit_v0.1 registered, path=./plugins\bdunit
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin CustomDifyApp_v0.2 registered, path=./plugins\custom_dify_app
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Dungeon_v1.0 registered, path=./plugins\dungeon
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Finish_v1.0 registered, path=./plugins\finish
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Godcmd_v1.0 registered, path=./plugins\godcmd
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Hello_v0.1 registered, path=./plugins\hello
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin JinaSum_v0.0.1 registered, path=./plugins\jina_sum
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Keyword_v0.1 registered, path=./plugins\keyword
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin linkai_v0.1.0 registered, path=./plugins\linkai
[INFO][2025-03-02 09:12:16][plugin_manager.py:41] - Plugin Role_v1.0 registered, path=./plugins\role
[WARNING][2025-03-02 09:12:16][plugin_manager.py:113] - Failed to import plugin tool: No module named 'chatgpt_tool_hub'
[DEBUG][2025-03-02 09:12:16][plugin_manager.py:185] - plugins.json config={'plugins': SortedDict({'Godcmd': {'enabled': True, 'priority': 999}, 'Keyword': {'enabled': True, 'priority': 900}, 'Banwords': {'enabled': False, 'priority': 100}, 'linkai': {'enabled': False, 'priority': 99}, 'JinaSum': {'enabled': False, 'priority': 10}, 'Role': {'enabled': False, 'priority': 0}, 'Dungeon': {'enabled': False, 'priority': 0}, 'CustomDifyApp': {'enabled': True, 'priority': 0}, 'BDunit': {'enabled': False, 'priority': 0}, 'Hello': {'enabled': True, 'priority': -1}, 'Finish': {'enabled': True, 'priority': -999}}, sort_func=<lambda>, reverse=True)}
[DEBUG][2025-03-02 09:12:16][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\godcmd\config.json, exist=True
[DEBUG][2025-03-02 09:12:16][plugin.py:28] - loading plugin config, plugin_name=Godcmd, conf={'password': '', 'admin_users': []}
[INFO][2025-03-02 09:12:16][godcmd.py:194] - [Godcmd] 因未设置口令，本次的临时口令为7586。
[INFO][2025-03-02 09:12:16][godcmd.py:210] - [Godcmd] inited
[DEBUG][2025-03-02 09:12:16][keyword.py:35] - [keyword]加载配置文件D:\Program\dify-on-wechat\plugins\keyword\config.json
[INFO][2025-03-02 09:12:16][keyword.py:41] - [keyword] {}
[INFO][2025-03-02 09:12:16][keyword.py:43] - [keyword] inited.
[DEBUG][2025-03-02 09:12:16][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\custom_dify_app\config.json, exist=False
[DEBUG][2025-03-02 09:12:16][plugin.py:28] - loading plugin config, plugin_name=CustomDifyApp, conf=None
[INFO][2025-03-02 09:12:16][custom_dify_app.py:25] - [CustomDifyApp] config is None
[DEBUG][2025-03-02 09:12:16][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\hello\config.json, exist=False
[DEBUG][2025-03-02 09:12:16][plugin.py:28] - loading plugin config, plugin_name=Hello, conf=None
[DEBUG][2025-03-02 09:12:16][hello.py:126] - No Hello plugin config.json, use plugins/hello/config.json.template
[INFO][2025-03-02 09:12:16][hello.py:38] - [Hello] inited
[INFO][2025-03-02 09:12:16][finish.py:23] - [Finish] inited
[INFO][2025-03-02 09:12:16][gewechat_channel.py:89] - [gewechat] sleep 3 seconds waiting for server to start, then set callback
[INFO][2025-03-02 09:12:16][gewechat_channel.py:107] - [gewechat] start callback server: http://192.168.31.235:9919/v2/api/callback/collect, using port 9919
[DEBUG][2025-03-02 09:12:19][gewechat_channel.py:229] - [gewechat] receive data: b'{"testMsg":"\xe5\x9b\x9e\xe8\xb0\x83\xe5\x9c\xb0\xe5\x9d\x80\xe9\x93\xbe\xe6\x8e\xa5\xe6\x88\x90\xe5\x8a\x9f\xef\xbc\x81","token":"934c7ad0b1994c13a52374f5be9f39fb"}'
[DEBUG][2025-03-02 09:12:19][gewechat_channel.py:234] - [gewechat] 收到gewechat服务发送的回调测试消息
[DEBUG][2025-03-02 09:16:50][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:16:50][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:16:50][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:16:51][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:16:51][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:16:51][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:16:57][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:16:57][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:16:57][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:16:57][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:16:57][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:16:57][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:16:57][audio_convert.py:10] - import pysilk failed, wechaty voice message will not be supported.
[WARNING][2025-03-02 09:16:57][audio_convert.py:20] - import pilk failed, silk voice conversion will not be supported. Try: pip install pilk
[INFO][2025-03-02 09:16:57][gewechat_channel.py:62] - [gewechat] init: base_url: http://192.168.31.235:2531/v2/api, token: 934c7ad0b1994c13a52374f5be9f39fb, app_id: wx_h_wNG48atr8kFFQWkELQ2, download_url: http://192.168.31.235:2532/download
[INFO][2025-03-02 09:16:57][plugin_manager.py:50] - Loading plugins config...
[INFO][2025-03-02 09:16:57][plugin_manager.py:88] - Scaning plugins ...
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Banwords_v1.0 registered, path=./plugins\banwords
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin BDunit_v0.1 registered, path=./plugins\bdunit
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin CustomDifyApp_v0.2 registered, path=./plugins\custom_dify_app
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Dungeon_v1.0 registered, path=./plugins\dungeon
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Finish_v1.0 registered, path=./plugins\finish
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Godcmd_v1.0 registered, path=./plugins\godcmd
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Hello_v0.1 registered, path=./plugins\hello
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin JinaSum_v0.0.1 registered, path=./plugins\jina_sum
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Keyword_v0.1 registered, path=./plugins\keyword
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin linkai_v0.1.0 registered, path=./plugins\linkai
[INFO][2025-03-02 09:16:57][plugin_manager.py:41] - Plugin Role_v1.0 registered, path=./plugins\role
[WARNING][2025-03-02 09:16:57][plugin_manager.py:113] - Failed to import plugin tool: No module named 'chatgpt_tool_hub'
[DEBUG][2025-03-02 09:16:57][plugin_manager.py:185] - plugins.json config={'plugins': SortedDict({'Godcmd': {'enabled': True, 'priority': 999}, 'Keyword': {'enabled': True, 'priority': 900}, 'Banwords': {'enabled': False, 'priority': 100}, 'linkai': {'enabled': False, 'priority': 99}, 'JinaSum': {'enabled': False, 'priority': 10}, 'Role': {'enabled': False, 'priority': 0}, 'Dungeon': {'enabled': False, 'priority': 0}, 'CustomDifyApp': {'enabled': True, 'priority': 0}, 'BDunit': {'enabled': False, 'priority': 0}, 'Hello': {'enabled': True, 'priority': -1}, 'Finish': {'enabled': True, 'priority': -999}}, sort_func=<lambda>, reverse=True)}
[DEBUG][2025-03-02 09:16:57][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\godcmd\config.json, exist=True
[DEBUG][2025-03-02 09:16:57][plugin.py:28] - loading plugin config, plugin_name=Godcmd, conf={'password': '', 'admin_users': []}
[INFO][2025-03-02 09:16:57][godcmd.py:194] - [Godcmd] 因未设置口令，本次的临时口令为2658。
[INFO][2025-03-02 09:16:57][godcmd.py:210] - [Godcmd] inited
[DEBUG][2025-03-02 09:16:57][keyword.py:35] - [keyword]加载配置文件D:\Program\dify-on-wechat\plugins\keyword\config.json
[INFO][2025-03-02 09:16:57][keyword.py:41] - [keyword] {}
[INFO][2025-03-02 09:16:57][keyword.py:43] - [keyword] inited.
[DEBUG][2025-03-02 09:16:57][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\custom_dify_app\config.json, exist=False
[DEBUG][2025-03-02 09:16:57][plugin.py:28] - loading plugin config, plugin_name=CustomDifyApp, conf=None
[INFO][2025-03-02 09:16:57][custom_dify_app.py:25] - [CustomDifyApp] config is None
[DEBUG][2025-03-02 09:16:57][plugin.py:21] - loading plugin config, plugin_config_path=./plugins\hello\config.json, exist=False
[DEBUG][2025-03-02 09:16:57][plugin.py:28] - loading plugin config, plugin_name=Hello, conf=None
[DEBUG][2025-03-02 09:16:57][hello.py:126] - No Hello plugin config.json, use plugins/hello/config.json.template
[INFO][2025-03-02 09:16:57][hello.py:38] - [Hello] inited
[INFO][2025-03-02 09:16:57][finish.py:23] - [Finish] inited
[INFO][2025-03-02 09:16:57][gewechat_channel.py:89] - [gewechat] sleep 3 seconds waiting for server to start, then set callback
[INFO][2025-03-02 09:16:57][gewechat_channel.py:107] - [gewechat] start callback server: http://192.168.31.235:9919/v2/api/callback/collect, using port 9919
[DEBUG][2025-03-02 09:17:00][gewechat_channel.py:229] - [gewechat] receive data: b'{"testMsg":"\xe5\x9b\x9e\xe8\xb0\x83\xe5\x9c\xb0\xe5\x9d\x80\xe9\x93\xbe\xe6\x8e\xa5\xe6\x88\x90\xe5\x8a\x9f\xef\xbc\x81","token":"934c7ad0b1994c13a52374f5be9f39fb"}'
[DEBUG][2025-03-02 09:17:00][gewechat_channel.py:234] - [gewechat] 收到gewechat服务发送的回调测试消息
[DEBUG][2025-03-02 09:31:21][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:31:21][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:31:21][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:31:21][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:31:21][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:31:21][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:31:46][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:31:46][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:31:46][config.py:267] - [Config] User datas loaded.
[DEBUG][2025-03-02 09:31:46][config.py:338] - [INIT] set log level to DEBUG
[INFO][2025-03-02 09:31:46][config.py:340] - [INIT] load config: {'channel_type': 'gewechat', 'coze_api_base': 'https://api.coze.cn', 'coze_api_key': 'pat*****mRp', 'coze_bot_id': '7472418615191896074', 'debug': '1', 'dify_api_base': 'http://localhost/v1', 'dify_api_key': 'app*****91b', 'dify_app_type': 'chatbot', 'gewechat_app_id': 'wx_h_wNG48atr8kFFQWkELQ2', 'gewechat_base_url': 'http://192.168.31.235:2531/v2/api', 'gewechat_callback_url': 'http://192.168.31.235:9919/v2/api/callback/collect', 'gewechat_download_url': 'http://192.168.31.235:2532/download', 'gewechat_token': '934c7ad0b1994c13a52374f5be9f39fb', 'message_buffer_second': 5, 'model': 'coze', 'nick_name_black_list': [], 'sikulix_describe': 'wecom', 'wx_name': '望龙（ai销售版）', 'wxid_black_list': [], 'wxid_recip_note': ''}
[INFO][2025-03-02 09:31:46][config.py:267] - [Config] User datas loaded.
[INFO][2025-03-02 09:31:53][config.py:353] - [Config] Configuration saved.
